dmscripts/send_stats_to_performance_platform.py

```diff
@@ -139,10 +139,14 @@ def send_by_stage_stats(stats, timestamp_string, period, pp_bearer, pp_service):
         "period": period
     } for stage in processed_stats]
 
+    if dry_run:
+        print(f"Would send the following stage stats to Performance Platform API:")
```

Comment:
There _is_ a configured `logger` in this module, y'know...

Reply:
There is indeed! The other `print` statement was wrong anyway - have pushed a commit to fix.